build.gradle

```diff
@@ -95,7 +95,7 @@ subprojects {
     }
 
     ext {
-        releasable = true // all corda-api jars are for external consumption
+        releasable = true // all corda-api jars are for external consumable
```

Comment:
rephrase: 
```all corda-api jars are externally consumable```

Reply:
Thanks @yarivus  .Updated + some other tidy up (For clarity making property names and task names non the same value 'resolvable' ) and ensure properties are resolved as Booleans rather than stings.